src/framework/components/anim/state-graph.js

```diff
@@ -1,23 +1,46 @@
 function AnimStateGraph(data) {
     this._layers = [];
     this._parameters = {};
-    if (data) {
-        if (data.states) {
-            this._layers = [];
-            this._layers.push({
-                name: 'Base',
-                states: data.states,
-                transitions: data.transitions
-            });
-        } else {
-            this._layers = data.layers;
-        }
-        this._parameters = {};
-        for (var paramId in data.parameters) {
-            var param = data.parameters[paramId];
-            this._parameters[param.name] = { type: param.type, value: param.value };
+    var i;
```

Comment:
Might be good to document what data is in this case?

Reply:
👍